src/detect-engine-register.h

```diff
@@ -86,11 +83,26 @@ enum DetectKeywordId {
     DETECT_BYTETEST,
     DETECT_BYTEJUMP,
     DETECT_BYTEMATH,
+    DETECT_BYTE_EXTRACT,
+    DETECT_DATASET,
+    DETECT_DATAREP,
+    DETECT_BASE64_DECODE,
+    DETECT_BASE64_DATA,
+    DETECT_BSIZE,
+    DETECT_ASN1,
```

Comment:
Why is ASN1 here ?

Reply:
it's a content inspect keyword, like the rest of the list

Reply:
Ok, I see `git grep DETECT_ASN src/detect-engine-content-inspection.c`